kolena/_utils/state.py

```diff
@@ -129,21 +119,17 @@ def get_client_state() -> _ClientState:
     return CLIENT_STATE.get(_client_state)
 
 
-def is_client_initialized() -> bool:
-    try:
-        get_client_state().assert_initialized()
-    except (InvalidClientStateError, UninitializedError):
-        return False
-    return True
+def is_client_uninitialized() -> bool:
+    return get_client_state().jwt_token is None
 
 
 def kolena_initialized(func: Callable) -> Callable:
-    """Raises InvalidKolenaStateError if not initialized"""
+    """Attempts to initialize client if not initialized"""
 
     @functools.wraps(func)
     def wrapper(*args: Any, **kwargs: Any) -> Any:
-        client_state = get_client_state()
-        client_state.assert_initialized()
+        if is_client_uninitialized():
+            kolena.initialize(verbose=True)
```

Comment:
Should we use `verbose=True` here?

Reply:
Ya I think we should use default args. Will update